core/hotspot/slot.go

```diff
@@ -15,10 +15,13 @@
 package hotspot
 
 import (
+	"context"
 	"time"
 
+	"golang.org/x/sync/errgroup"
+	"gopkg.in/errgo.v2/fmt/errors"
```

Comment:
Maybe `github.com/pkg/errors`  is enough?

Reply:
yes, it's enough ,I will replace it

Reply:
> Could you please provide some benchmark for involving errgroup?

ok ,i will provide it today

Reply:
Nice~